### PR TITLE
Movie list wm

### PR DESCRIPTION
**Pull request by WoytekMig**
First page of MoviesList is generating, and it's responsive. Rest of the pages in next PR, when I'll add pagination ;)

I'm open for new ideas to resolve warning in generating url to Movie poster ;) This one works fine, but well.. :D

### CODE REVIEW

File: src/features/MoviesList/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
```
Comment (Arcziwald) [inline src/features/MoviesList/index.js]: something is not used here :-)

Comment (WoytekMig) [reply src/features/MoviesList/index.js]: that's why this all resume-thing is so important  ;) I missed it - already repaired ! :)

**Review by Arcziwald — APPROVED** [review]

File: src/features/MoviesList/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                            tag={element.genre_ids}
+                        />
+                    </React.Fragment>
+                )))}
+
+        </MainContainer>
+        </>
+    )
+
+};
```
Comment (KrystianZybura) [inline src/features/MoviesList/index.js]: Code doesn't seem to be formatted here. I highly recommend installing prettier extension for VSC and setting VSC formatting on file save 💪

File: src/common/MovieTile/styled.js [hunk 3]
```diff
@@ -3,27 +3,33 @@ import { ReactComponent as YellowStar } from "../../images/shape-star.svg";
 
 export const Wrapper = styled.div`
     max-width: 324px;
-    height: 650px;
+   /*  min-height: 650px; */
```
Comment (KrystianZybura) [inline src/common/MovieTile/styled.js]: I don't think this comment is needed here 👌

**Review by KrystianZybura — COMMENTED** [review]
Great job, just few simple things 😎

File: src/features/MoviesList/useMoviesData.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react";
+
+const nowPlaying = "https://api.themoviedb.org/3/movie/now_playing?language=en-US&page=1&region=pl";
```
Comment (KrystianZybura) [inline src/features/MoviesList/useMoviesData.js]: I don't think it's being used anywhere.. 😅

**Review by KrystianZybura — COMMENTED** [review]
One more thing 👌

File: src/features/MoviesList/index.js (hunk 2 above)
Comment (WoytekMig) [reply src/features/MoviesList/index.js]: interesting - I already got that set :D 
But what should be different ? :)

File: src/common/MovieTile/styled.js (hunk 3 above)
Comment (WoytekMig) [reply src/common/MovieTile/styled.js]: got me ! I forgot to deleted it :) 

already done !

File: src/features/MoviesList/useMoviesData.js (hunk 4 above)
Comment (WoytekMig) [reply src/features/MoviesList/useMoviesData.js]: indeed... :D 

deleted !

**Review by KrystianZybura — APPROVED** [review]

**Pull request merged**